build/templates/und.json

```diff
@@ -0,0 +1,21 @@
+{
+  "metadata": {
+    "alphabet": "Zzzz",
```

Comment:
Is this valid?

Reply:
Yes. See http://www.unicode.org/reports/tr35/#Unknown_or_Invalid_Identifiers.